eth/beacon/helpers.py

```diff
@@ -1,17 +1,167 @@
 from typing import (
     Iterable,
+    Any,
     List,
+    Tuple,
     TYPE_CHECKING,
 )
 
 from eth_utils import (
     to_tuple,
 )
 
+from eth_typing import (
+    Hash32,
+)
+
+from eth.utils.blake import (
+    blake,
+)
+from eth.beacon.types.shard_and_committee import (
+    ShardAndCommittee,
+)
+
+
 if TYPE_CHECKING:
+    from eth.beacon.config import BeaconConfig  # noqa: F401
+    from eth.beacon.types.active_state import ActiveState  # noqa: F401
+    from eth.beacon.types.attestation_record import AttestationRecord  # noqa: F401
+    from eth.beacon.types.block import Block  # noqa: F401
+    from eth.beacon.types.crystallized_state import CrystallizedState  # noqa: F401
     from eth.beacon.types.validator_record import ValidatorRecord  # noqa: F401
 
 
+#
+# Get block hashes
+#
+def get_block_hash(
+        active_state: 'ActiveState',
+        current_block: 'Block',
+        slot: int,
+        beacon_config: 'BeaconConfig') -> Hash32:
```

Comment:
If #1340 is going to be prioritized soon, I have no issue leaving these in, but it may be prudent to go ahead and convert these functions to accept only the parameters from the `beacon_config` that they actually use.  Also, fine to clean it all up in one go in a subsequent PR

Reply:
For some functions, e.g., `get_block_hash` actually can just take `recent_block_hashes, current_block_slot_number, target_slot, cycle_length` as the parameters, `get_signed_parent_hashes` can just take `recent_block_hashes, block_slot_number, attestation, cycle_length` as parameters. I'd like to clean/slim them in this PR so that they will be easier to create tests.